src/components/ui/toolbox.ts

```diff
@@ -83,7 +91,7 @@ export default class Toolbox extends EventsDispatcher<ToolboxEventMap> {
   /**
    * Popover instance. There is a util for vertical lists.
    */
-  private popover: Popover | undefined;
+  private popover: PopoverDesktop | PopoverMobile | undefined;
```

Comment:
Might worth it to create a union type as `PopoverDesktop | PopoverMobile` used several times across the codebase

Reply:
Or just describe an interface with common props and methods

Reply:
done